src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonic.hpp

```diff
@@ -163,7 +163,7 @@ struct EvolutionMetavars {
         tmpl::list<StrahlkorperGr::Tags::AreaElement<Frame::Inertial>>;
   };
 
-  struct Horizon {
+  struct AhA {
```

Comment:
Not for this PR, but I generally never liked `AhA` and `AhB` because I never know which is the larger hole. Could we switch to something like `LargerAh` or `AhBigger` (and an opposite name) in the future? And then have `AhC` be `AhCommon` or `CommonAh`? I think that would help make the code a lot more accessible.

Reply:
Yes please :D Perhaps `Primary` and `Secondary`?

Reply:
Ok, I can change in whatever PR actually has two holes.  Note that for equal mass systems, either hole can be slightly larger than the other due to truncation error, so the names will not always be correct.

Reply:
Thanks! Yes, though personally rather have the name be incorrect by truncation error than be unsure about every simulation :)